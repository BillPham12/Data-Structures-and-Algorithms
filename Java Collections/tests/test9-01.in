hello
hello1
hello
mynameis
kongkrap
katulakelas
1231231312
333333333
4444444444
555555555
6666666666
