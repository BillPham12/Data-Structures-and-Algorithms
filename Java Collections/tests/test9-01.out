hello
hello1
mynameis
kongkrap
katulakelas
1231231312
333333333
4444444444
555555555
6666666666
